packages/fxa-payments-server/src/components/PaymentConsentCheckbox/index.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Note: test coverage for this component exists in
+ * components/PaymentForm/index.test.tsx under the
+ * title 'rendering the legal checkbox Localized component'
+ */
+
+import React, { useContext } from 'react';
```

Comment:
Nit: Doesn't look like we're using `React` explicitly here; so we don't need to import it.

Reply:
Declaring React like this is necessary whenever using JSX for the version we are using, if we ever update to React 17 we can leave these declarations out https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html

Reply:
Hm that's interesting thanks. We're using it now directly anyway in your latest patch for the onClick optional prop.